Header header
float64[] x
float64[] u
float64[] xNext